package.json

```diff
@@ -13,7 +13,7 @@
     "lint": "prettier-standard --lint '**/*'",
     "test-watch": "npm run lint && jest --onlyChanged --watch",
     "test": "npm run lint && jest",
-    "pre": "./timestamp.sh > versionInfo.js"
+    "pre": "./timestamp.sh > src/versionInfo.js"
```

Comment:
I'd leave this one in the top-level, since it's auto-generated rather than a source file we author.

Reply:
It's referred to in src/index.js, so I would prefer to have it in src folder - I at least don't want to have it on root, but could have another folder to put it in since it's a bit different than the source files in src